newIDE/app/src/ObjectsRendering/Renderers/RenderedCustomObjectInstance.js

```diff
@@ -161,21 +176,25 @@ export default class RenderedCustomObjectInstance extends RenderedInstance {
 
     this.childrenInstances = [];
     this.childrenRenderedInstances = eventBasedObject
-      ? mapFor(0, eventBasedObject.getObjectsCount(), i => {
+      ? mapReverseFor(0, eventBasedObject.getObjectsCount(), i => {
           const childObject = eventBasedObject.getObjectAt(i);
           const childObjectConfiguration = customObjectConfiguration.getChildObjectConfiguration(
             childObject.getName()
           );
           const childInstance = new ChildInstance();
           this.childrenInstances.push(childInstance);
-          return ObjectsRenderingService.createNewInstanceRenderer(
+          const renderer = ObjectsRenderingService.createNewInstanceRenderer(
             project,
             layout,
             // $FlowFixMe Use real object instances.
             childInstance,
             childObjectConfiguration,
             this._pixiObject
           );
+          if (renderer instanceof RenderedTextInstance) {
+            renderer._pixiObject.style.align = 'center';
```

Comment:
Could this be done as a property of the text object?

Reply:
Let's add a "TODO: this should be a property added to the text object. When this property is added, remove these special cases".

Or if it's really not too complex => add it to the text + TextObject editor.